presto-cli/src/main/resources/META-INF/native-image/resource-config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "resources":[
+    {"pattern":"\\QMETA-INF/services/org.jline.terminal.spi.JnaSupport\\E"},
+    {"pattern":"\\Qcom/sun/jna/darwin/libjnidispatch.jnilib\\E"},
+    {"pattern":"\\Qorg/jline/utils/capabilities.txt\\E"},
+    {"pattern":"\\Qorg/jline/utils/xterm-256color.caps\\E"},
+    {"pattern": "org/jline/utils/.*caps$"},
+    {"pattern": "org/jline/utils/capabilities\\.txt$"},
+    {"pattern": "com/amazonaws/.*json$"},
+    {"pattern": "com/amazonaws/global/handlers/request\\.handler2s"},
+      {"pattern": ".*properties$"},
+      {"pattern": "META-INF/native/windows64/jansi.dll"},
+      {"pattern":"META-INF/native/osx/libjansi.jnilib"},
+      {"pattern":"org/fusesource/jansi/META-INF/native/osx/libjansi.jnilib"},
+      {"pattern":"org/fusesource/jansi/jansi/META-INF/native/osx/libjansi.jnilib"},
+      {"pattern":"org/fusesource/jansi/jansi/jansi/META-INF/native/osx/libjansi.jnilib"},
+      {"pattern":".*/libjansi.jnilib$"},
+      {"pattern":"META-INF/native/osx/libjansi.jnilib"},
+      {"pattern":"com/facebook/presto/spi/type/zone-index.properties"},
+      {"pattern":"jline/console/completer/CandidateListCompletionHandler.properties"},
+      {"pattern":"META-INF/services/org.jline.terminal.spi.JansiSupport"},
+      {"pattern":"META-INF/services/org.jline.terminal.spi.JnaSupport"},
+      {"pattern":"logback.xml"},
```

Comment:
Where does this come from?

Reply:
I'll cleanup configs when I have working configuration on linux and windows as well